tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -84,6 +84,10 @@ setup() {
 }
 
 teardown() {
+	# The last test set the policy to deny all resources,
+	# so we need to set it to default for next tests
+	# (e.g k8s-sealed-secrets.bats)
```

Comment:
Sorry, maybe I'm missing something, but is there an argument to (also?) set the default policy in the setup of the sealed_secrets test if it requires that, in case we change the order in future?

Reply:
yep, I will relocate `kbs_set_default_policy()` from k8s-confidential-attestation.bats to k8s-sealed-secrets.bats. Thanks.